genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/ApplicationTask.java

```diff
@@ -70,6 +75,7 @@ public void executeTask(
         @NotNull
         final Map<String, Object> context
     ) throws GenieException, IOException {
+        final Map<String, String> tags = MetricsConstants.newSuccessTagsMap();
```

Comment:
This doesn't look like a "constant" to me. More like a utility method? Perhaps we need a MetricsUtils class.